jdks/download-jdks.sh

```diff
@@ -89,21 +90,25 @@ function unpack_mac_jdk {
         xar -xf /Volumes/JDK*/JDK*.pkg
         hdiutil detach /Volumes/JDK*
     else # Linux
-        7z x ../downloads/jdk-macosx-x64.dmg > /dev/null
+        7z x ../downloads/jdk-macosx-x64.dmg 
```

Comment:
Same here: Output isn't discarded and hence also destroys that ">> Building the Mac OSX JDK" flair, however as pointed out above, I might be overengineering things